Assets/Mirror/Examples/Lobbey/Scripts/NetworkLobbyPlayerExt.cs

```diff
@@ -0,0 +1,38 @@
+﻿using UnityEngine;
+using Mirror;
+using UnityEngine.SceneManagement;
+
```

Comment:
This should be in the namespace "Mirror.Examples.Lobby"

Reply:
fixed